Prefix/InfixToPrefixConverter.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Project2_Group_17.Prefix
+{
+    public class InfixToPrefixConverter
+    {
```

Comment:
I decided to make everything static since there wasn't any need for any state or anything. Do you agree with that? If so I think any class that has only static methods should be static too

Reply:
Yeah that makes sense, no point in being able to declare it in its given state